src/main/java/uk/gov/ons/ssdc/supporttool/service/CaseService.java

```diff
@@ -89,6 +93,20 @@ public void buildAndSendUpdateSensitiveSampleEvent(
     pubSubTemplate.publish(topic, event);
   }
 
+  public void buildAndSendUpdateSampleEvent(UpdateSample updateSample, String userEmail) {
```

Comment:
what is this? why is it in case processor? update sample is an event which is published _to_ the case processor, not _from_ the case processor